cluster-sync/external/provider.sh

```diff
@@ -16,11 +16,27 @@ function verify() {
 
 
 function up() {
-  echo "using external provider"
+  echo "External provider"
 }
 
 function configure_storage() {
-  echo "Local storage not needed for external provider..."
+  if [ "${EXTERNAL_PROVIDER}" == "openshift" ]; then
```

Comment:
Why is this EXTERNAL_PROVIDER necessary to get hpp or ceph?

Reply:
Leftover I think, fixing.